{
	"cve": {
		"data_type": "CVE",
		"data_format": "MITRE",
		"data_version": "4.0",
		"CVE_data_meta": {
			"ID": "CVE-2020-25710",
			"ASSIGNER": "secalert@redhat.com"
		},
		"problemtype": {
			"problemtype_data": [
				{
					"description": []
				}
			]
		},
		"references": {
			"reference_data": [
				{
					"url": "https://lists.debian.org/debian-lts-announce/2020/12/msg00008.html",
					"name": "[debian-lts-announce] 20201204 [SECURITY] [DLA 2481-1] openldap security update",
					"refsource": "MLIST",
					"tags": []
				},
				{
					"url": "https://git.openldap.org/openldap/openldap/-/commit/ab3915154e69920d480205b4bf5ccb2b391a0a1f#a2feb6ed0257c21c6672793ee2f94eaadc10c72c",
					"name": "https://git.openldap.org/openldap/openldap/-/commit/ab3915154e69920d480205b4bf5ccb2b391a0a1f#a2feb6ed0257c21c6672793ee2f94eaadc10c72c",
					"refsource": "MISC",
					"tags": []
				},
				{
					"url": "https://www.debian.org/security/2020/dsa-4792",
					"name": "DSA-4792",
					"refsource": "DEBIAN",
					"tags": []
				},
				{
					"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1899678",
					"name": "https://bugzilla.redhat.com/show_bug.cgi?id=1899678",
					"refsource": "MISC",
					"tags": []
				}
			]
		},
		"description": {
			"description_data": [
				{
					"lang": "en",
					"value": "A flaw was found in OpenLDAP in versions before 2.4.56. This flaw allows an attacker who sends a malicious packet processed by OpenLDAP to force a failed assertion in csnNormalize23(). The highest threat from this vulnerability is to system availability."
				}
			]
		}
	},
	"configurations": {
		"CVE_data_version": "4.0",
		"nodes": []
	},
	"impact": {},
	"publishedDate": "2021-05-28T11:15Z",
	"lastModifiedDate": "2021-05-28T11:15Z"
}